tests/dummy/app/templates/nodes/detail/draft-registrations/detail.hbs

```diff
@@ -0,0 +1,71 @@
+
+<h2> {{model.draft.registrationSchema.name}} </h2>
+<p> Initiated by: {{model.draft.initiator.fullName}}</p>
+<p> Started: {{moment-format model.draft.datetimeInitiated}} </p>
+<p> Last Updated: {{moment-format model.draft.datetimeUpdated}} </p>
+
+<hr>
+
+<h3> Edit Draft Registration </h3>
+{{#each model.draft.registrationSchema.schema.pages as |page|}}
+
+    {{#each page.questions as |question|}}
+        <p> <label> {{question.title}} </label> </p>
+        {{#if question.description}}
+            <p> {{question.description}} </p>
+        {{/if}}
+        {{#if (eq question.type "choose")}}
+            {{#unless question.description}}
+                <p> <em> Please choose </em> </p>
+            {{/unless}}
+            <form>
+                {{#each question.options as |option|}}
+                    {{#if (eq (checkDataType option) "string")}}
+                        <p> <input onchange={{action "buildForm" value="target"}} type="radio" name={{question.qid}} value={{option}} checked={{selectedHelper option (draftTextInput model.draft.registrationMetadata question)}}> {{option}} </p>
```

Comment:
I won't be looking to block this PR based on UI code, but I'd be happy to pair sometime to look over alternate ways of dealing with UI. 

The use of nested helpers and validators makes it a bit hard to follow the intended logic here, but there are some alternative approaches.